package.json

```diff
@@ -14,10 +14,12 @@
     "moment": "^2.24.0",
     "node-sass": "^4.13.1",
     "prop-types": "^15.7.2",
+    "rc-slider": "^9.2.2",
     "react": "^16.12.0",
     "react-audio-player": "^0.11.1",
     "react-autosize-textarea": "^7.0.0",
     "react-avatar-edit": "^0.8.3",
```

Comment:
I think we can remove avatar-edit if it's not used anymore :)

Reply:
YEs we can :)